ouroboros-consensus-protocol/src/Ouroboros/Consensus/Protocol/TPraos.hs

```diff
@@ -531,6 +533,18 @@ instance SL.PraosCrypto c => PraosProtocolSupportsNode (TPraos c) where
         , ticknStatePrevHashNonce
         } = csTickn
 
+  getOpCertCounters _prx cdst = opcertCounters
+    where
+      TPraosState { tpraosStateChainDepState } = cdst
+      SL.ChainDepState {
+        SL.csProtocol
+        } = tpraosStateChainDepState
+      SL.PrtclState
```

Comment:
I'd generally prefer record syntax for this pattern, but the field we want has a unique type, so no real risk here.